src/cmd-buildextend-installer

```diff
@@ -145,6 +152,19 @@ def cp_reflink(src, dest):
     subprocess.check_call(['cp', '--reflink=auto', src, dest])
 
 
+# Make stream hash for `rdcore stream-hash`
+def make_stream_hash(src, dest):
```

Comment:
Any reason you implemented this in Python instead of `rdcore` as well so both reading and writing the hash file sit together? Plus, we're sure to use OpenSSL for hashing, whereas... briefly looking at the Python `hashlib` docs and the CPython source, it seems like it uses its own C implementation for SHA-256. How's the speed?

Cool as is too of course (and we can change it in a follow-up as well).

Though maybe once we [document the format](https://github.com/coreos/coreos-installer/pull/305#discussion_r460056814) a bit, let's add a permalink to it here?

Reply:
> Any reason you implemented this in Python instead of `rdcore` as well so both reading and writing the hash file sit together?

My initial instinct was to put it in `rdcore`, but the additional build-time dependency seemed more troublesome than the 10 lines of Python.  But yeah, we can always change it later.  I'll add a link to the format docs.

> Plus, we're sure to use OpenSSL for hashing, whereas... briefly looking at the Python hashlib docs and the CPython source, it seems like it uses its own C implementation for SHA-256. How's the speed?

I'm seeing 347 MiB/sec for this code (using `timeit`) and 335 MiB/sec for `rdcore stream-hash` (including process startup overhead).